src/celeritas/em/params/WentzelOKVIParams.cc

```diff
@@ -179,772 +181,1241 @@ real_type WentzelOKVIParams::calc_nuclear_form_prefactor(IsotopeView const& iso)
 
 //---------------------------------------------------------------------------//
 /*!
- * Get interpolated Mott coefficients for an element.
+ * Get interpolated Mott coefficients for electrons for an element.
  *
- * These coefficients are used by the Lijian, Quing, Zhengming
- * expression [PRM 8.48] for atomic numbers 1 <= Z <= 92.
- * This data was taken from Geant4's G4MottData.hh file.
+ * These coefficients are for electrons interacting with nuclei with 1 <= Z <=
+ * 118 and taken from table 3 in M.J. Boschini et al, Radiat. Phys. Chem. 90
+ * (2013), 39-66. Geant4 provides the same data in G4MottData.hh, but only for
+ * 1 <= Z <= 92.
+ */
+auto WentzelOKVIParams::get_electron_mott_coeffs(AtomicNumber z) -> CoeffMat
+{
+    CELER_EXPECT(z);
+
+    // clang-format off
+    static CoeffMat const electron_mott_coeffs[]
+        = {{{{1.00000e+00, 2.67363e-08, 7.11530e-08, -9.77030e-08, -6.69132e-07, -3.09263e-07},
```

Comment:
I could also format this differently...

Reply:
I think the formatting is fine! It's not like they're readable anyway...